src/transformers/models/xlm_prophetnet/__init__.py

```diff
@@ -15,20 +15,49 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import TYPE_CHECKING
 
-from ...file_utils import is_sentencepiece_available, is_torch_available
-from .configuration_xlm_prophetnet import XLM_PROPHETNET_PRETRAINED_CONFIG_ARCHIVE_MAP, XLMProphetNetConfig
+from ...file_utils import _LazyModule, is_sentencepiece_available, is_torch_available
 
 
+_import_structure = {
+    "configuration_xlm_prophetnet": [
+        "XLM_PROPHETNET_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "XLMProphetNetConfig",
```

Comment:
```suggestion
        "XLM_PROPHETNET_PRETRAINED_CONFIG_ARCHIVE_MAP", "XLMProphetNetConfig",
```
Nit, it probably fits in one line.